include/flatcc/flatcc_rtconfig.h

```diff
@@ -63,6 +63,14 @@ extern "C" {
 #define FLATCC_TRACE_VERIFY 0
 #endif
 
+/*
+ * Some producers allow empty vectors to be misaligned.
+ * The following setting will cause the verifier to check for an
+ * empty vector before checking alignment of the vector's elements.
+ */
+#if !defined(FLATCC_TOLERATE_MISALIGNED_EMPTY_VECTORS)
+#define FLATCC_TOLERATE_MISALIGNED_EMPTY_VECTORS 0
+#endif
 
```

Comment:
I think we may have to set this to 1 and flip the CMake setting.

Reply:
In that case, I should probably rename the option to `FLATCC_REFUSE_MISALIGNED_EMPTY_VECTORS` and leave *its* default at 0

Reply:
FLATCC_ENFORCE_ALIGNED_EMPTY_VECTORS